imageio/plugins/pyav.py

```diff
@@ -0,0 +1,839 @@
+"""Read/Write Videos (and images) using PyAV.
+
+Backend Library: `PyAV <https://pyav.org/docs/stable/>`_
+
+PyAVPlugin is a plugin that wraps pyAV. pyAV is a set of pythonic bindings for
+the FFMPEG library. As such this plugin is similar our famous FFMPEG plugin, but
+offers nicer bindings and aims to superseed it in the future.
+
+"""
+
+from math import ceil
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import av
+import av.filter
+import numpy as np
+from numpy.typing import ArrayLike
```

Comment:
The typing module is really new. Do you need to bump numpy quite high?

Reply:
Yes, `ArrayLike` was introduced in numpy 1.20 and we depend on that since ImageIO v2.15.

Reply:
I'm actually now debating if it would make sense to move all the type annotations into a separate repo and to relax the numpy dependency. Not 100% sure if we depend on anything else from `numpy>=1.20`...

Reply:
I created https://github.com/imageio/imageio/issues/763 to keep track of this. Since we are using `ArrayLike` in other places in the library this is a larger refactor and I will deal with this in a separate PR.